dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -413,10 +414,15 @@ public Map<String, Object> updateUser(User loginUser, int userId,
         }
 
         if (StringUtils.isNotEmpty(userPassword)) {
```

Comment:
remove the outer not empty validate

Reply:
> remove the outer not empty validate

I found that my conditional judgment was wrong. I was very confused about the other judgment. I didn't rule out null values, but I didn't modify it to maintain consistency

Reply:
> > remove the outer not empty validate
> 
> I found that my conditional judgment was wrong. I was very confused about the other judgment. I didn't rule out null values, but I didn't modify it to maintain consistency

Sorry @syyangs799 , I found that my original comment is wrong, the outer non-null judgment needs to be retained, and will only be replaced when the new password is not null.
In this way, it's enough to keep the `checkPasswordLength` validate, what do you think?

Reply:
> > > remove the outer not empty validate
> > 
> > 
> > I found that my conditional judgment was wrong. I was very confused about the other judgment. I didn't rule out null values, but I didn't modify it to maintain consistency
> 
> Sorry @syyangs799 , I found that my original comment is wrong, the outer non-null judgment needs to be retained, and will only be replaced when the new password is not null. In this way, it's enough to keep the `checkPasswordLength` validate when password not empty, what do you think?

ok，I understand that this is to modify the user's other non password information